core/src/main/java/tachyon/worker/TachyonWorker.java

```diff
@@ -144,8 +151,11 @@ public static void main(String[] args) throws UnknownHostException {
           + "tachyon.Worker [<MasterHost:Port>]");
       System.exit(-1);
     }
-    
+
     TachyonConf tachyonConf = new TachyonConf();
+    if (args.length == 1) {
+      getMasterLocation(args[0], tachyonConf);
```

Comment:
Would it make more sense to take the conf.set parts out of the function and instead set it here? Since the method is a get, its kind of weird to mutate the conf and do nothing with the return value? Alternatively it can be back to setMasterLocation and return void?